ramls/schemas/SamlCheck.json

```diff
@@ -1,8 +1,10 @@
 {
   "$schema": "http://json-schema.org/draft-03/schema#",
   "type": "object",
+  "description": "Indicates whether or note SAML is configured and active",
```

Comment:
Either remove "or note" (I prefer this) or change "note" to "not". The same 3 lines below.